src/main.rs

```diff
@@ -193,4 +201,30 @@ fn main() {
                 exit(-6);
             });
     }
+
+    if !no_copy_lock {
+        info!("Transferring Cargo.lock file back to client.");
+        Command::new("rsync")
+            .arg("-a")
+            .arg("--delete")
+            .arg("--compress")
+            .arg("--info=progress2")
+            .arg(format!("{}:{}/Cargo.lock", build_server, build_path))
+            .arg(format!("{}/Cargo.lock", project_dir.to_string_lossy()))
+            .stdout(Stdio::inherit())
+            .stderr(Stdio::inherit())
+            .stdin(Stdio::inherit())
+            .output()
+            .unwrap_or_else(|e| {
+                error!(
+                    "Failed to transfer Cargo.lock back to local machine (error: {})",
+                    e
+                );
+                exit(-7);
+            });
+    }
+
+    if !output.status.success() {
+        exit(output.status.code().unwrap_or(0))
```

Comment:
```suggestion
        exit(output.status.code().unwrap_or(1))
```

You should default to `1` when the command did not execute successfully.